### ISSUE TITLE
Previews taking a lot of disk space

### ISSUE THREAD

**Issue by Cactucs** [avocode/avocode#705]
The previews are taking in my case 1 GB of strage (which is quite a lot of 32 GB SSD). 

Ideas:
 - Download only new ones - other download "on the go"
 - Option to disable previews

**Comment by jankuca** [comment]
Hello,

thanks for your feedback. We know about the issue and will have it fixed for the next release which should be out next week. The app will only be downloading previews for synced designs and the rest will be fetched on-demand from the server when needed. We are also considering different compression options to reduce the disk space demands.

If you delete the downloaded previews from the disk (`~/.avcd/userdata/$ID/previews`), the app will not be redownloading them. So if you need the space and would be willing to *temporarily* sacrifice the preview function, go ahead. We will tell you how to reset this and get it working again when the fix lands.

**Comment by Cactucs** [comment]
:+1:

**Comment by Cactucs** [comment]
Fixed in newest version, is it?

After update: "Preview image load error."

**Comment by jankuca** [comment]
Hello, yes, we changed the behaviour. The app now only downloads previews of designs you actually use; the rest is being fetched from the server on-demand. To reset the preview cache, use the following hidden command which we added:

- Press `Cmd+Shift+P` (`Ctrl+Shift+P` on Windows); the command palette pops up.
- Type in "Clear Previews".
- Click on the found result "Dev: Clear Previews" below the text field.

Let me know if this worked fine.
